index.html

```diff
@@ -2,9 +2,9 @@
 <html>
     <head>
         <meta charset="utf-8" />
-        <meta http-equiv="refresh" content="0; url=https://join.slack.com/t/crossplane/shared_invite/zt-26kxm0k2a-esMwXvMpfw8glyoMzukAlQ" />
+        <meta http-equiv="refresh" content="0; url=https://join.slack.com/t/crossplane/shared_invite/zt-29oiqovki-WuHqY~LUaAIHZVChxJNUqg" />
     </head>
     <body>
         <p><a href="https://join.slack.com/t/crossplane/shared_invite/zt-26kxm0k2a-esMwXvMpfw8glyoMzukAlQ">Redirecting you to a Crossplane Slack invitation</a></p>
     </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
Made the changes via GitHub UI and this line changed without me changing it. I can revert if we don't want it.

Reply:
no problems with keeping it!